application/src/main/java/org/thingsboard/server/service/install/update/DefaultDataUpdateService.java

```diff
@@ -189,6 +193,14 @@ public void updateData(String fromVersion) throws Exception {
                 } else {
                     log.info("Skipping audit logs migration");
                 }
+                boolean skipEdgeEventsMigrationTemp = getEnv("TB_SKIP_EDGE_EVENTS_MIGRATION", false);
+                if (!skipEdgeEventsMigrationTemp) {
+                    log.info("Updating data from version 3.4.1 to 3.4.2 ...");
```

Comment:
this line is going to duplicate with previous 'if (!skipAuditLogsMigration)' statetement
please refactor to check skipAuditLogsMigration and skipEdgeEventsMigrationTemp
if any 'true' - print this line single time